recipes/libflint/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.5.3.dev0" %}
+{% set commit = "6e1c767c29b5d1c0e94d33512181ad63e73f5fc7" %}
+
+package:
+  name: libflint
+  version: {{ version }}
+
+source:
+  fn: {{ commit }}.tar.gz
+  url: https://github.com/wbhart/flint2/archive/{{ commit }}.tar.gz
```

Comment:
This now appears to be present in the non-forked version. Could we please switch to the `fredrik-johansson` user or is there some other reason not to?